app/controllers/anonymous_feedback/problem_reports_controller.rb

```diff
@@ -0,0 +1,65 @@
+require "gds_api/support_api"
+
+class AnonymousFeedback::ProblemReportsController < AuthorisationController
+  def index
+    authorize! :request, :review_feedback
+
+    api_response = fetch_problem_reports
+
+    @feedback = AnonymousFeedbackPresenter.new(api_response)
+    @dates = present_date_filters(api_response)
+  end
+
+  def review
+    authorize! :request, :review_feedback
+
+    if review_params.any? && reviewed_items_successfully?
+      redirect_to anonymous_feedback_problem_reports_path(params.slice(:to_date, :from_date, :include_reviewed))
+    else
+      flash.now[:alert] = 'Something went wrong with this review'
+      render :index, status: 400
+    end
+  end
+
+  private
+
+  def fetch_problem_reports
+    AnonymousFeedbackApiResponse.new(
+      support_api.problem_reports(api_params).to_hash
+    )
+  end
+
+  def api_params
+    {
+      exclude_reviewed: !index_params[:include_reviewed],
```

Comment:
When I made the endpoint in the support api my thought process was that by default we would return everything (i.e. all reviewed and not reviewed), and then the various options filter that into subsets.

The front end is the other way around of course, so that only non-reviewed reports should be returned by default. Do you think it would be better for me to align this with the API?